frontend/e2e/infrastructure.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { expect, ftlTest } from './ftl-test'
+
+ftlTest('shows infrastructure', async ({ page }) => {
+  const infrastructureNavItem = page.getByRole('link', { name: 'Infrastructure' })
+  await infrastructureNavItem.click()
+  await expect(page).toHaveURL(/\/infrastructure$/)
+
+    const controllersTab = await page.getByRole('button', { name: 'Controllers' });
```

Comment:
Looks like the tabbing got a little messed up here?

Reply:
Ah yeah, `biome` was ignoring the `e2e` folder. Fixed!